docs/reference/ml/apis/get-datafeed-stats.asciidoc

```diff
@@ -86,8 +86,7 @@ The API returns the following results:
         "transport_address": "127.0.0.1:9300",
         "attributes": {
           "ml.machine_memory": "17179869184",
-          "ml.max_open_jobs": "20",
-          "ml.enabled": "true"
```

Comment:
This attribute was already removed in 7.0.0, but left in this one place in the docs by mistake